nifi-nar-bundles/nifi-kite-bundle/nifi-kite-processors/src/main/java/org/apache/nifi/processors/kite/AbstractKiteProcessor.java

```diff
@@ -47,33 +47,16 @@
 abstract class AbstractKiteProcessor extends AbstractProcessor {
 
     private static final Splitter COMMA = Splitter.on(',').trimResults();
-    protected static final Validator FILES_EXIST = new Validator() {
-        @Override
-        public ValidationResult validate(String subject, String configFiles,
-                ValidationContext context) {
-            if (configFiles != null && !configFiles.isEmpty()) {
-                for (String file : COMMA.split(configFiles)) {
-                    ValidationResult result = StandardValidators.FILE_EXISTS_VALIDATOR
-                            .validate(subject, file, context);
-                    if (!result.isValid()) {
-                        return result;
-                    }
-                }
-            }
-            return new ValidationResult.Builder()
-                    .subject(subject)
-                    .input(configFiles)
-                    .explanation("Files exist")
-                    .valid(true)
-                    .build();
-        }
-    };
 
     protected static final PropertyDescriptor CONF_XML_FILES
             = new PropertyDescriptor.Builder()
-            .name("Hadoop configuration files")
-            .description("A comma-separated list of Hadoop configuration files")
-            .addValidator(FILES_EXIST)
+            .name("hadoop-configuration-resources")
```

Comment:
I'd recommend we leave the name the same since changing the name will cause the processor to go invalid when someone upgrades an existing flow. 

We can still introduce displayName if you'd like to call "Hadoop configuration Resources" as opposed to "Hadoop configuration files", but we'd leave name as "Hadoop configuration files".

Reply:
Good point, fixed